### DISCUSSION TITLE
Unknown Symbol: LOCAL_LINK_SHORT1

### DISCUSSION THREAD

**Discussion by pmamat** [rspamd/rspamd#4970] [Q&A] [answered]
Hi, I am analyzing why some messages we are sending are marked as spam. 

In the _Symbols_ there are some entries with positive values, for some of which I found explanations on the net, others are not known by Google and other search engines:

```
 Symbol: ZERO_FONT(1.00)
 Symbol: LOCAL_LINK_SHORT1(2.30)
 Symbol: FORGED_SENDER(0.30)
 Symbol: AUTOGEN_PHP_SPAMMY(1.00)
 Symbol: MANY_INVISIBLE_PARTS(1.00)
```

I wonder especially about the `LOCAL_LINK_SHORT1` symbol. Could somebody please give me a hint, what is causing this?

What I found out is:
* `ZERO_FONT` and `MANY_INVISIBLE_PARTS` is caused by MJML
* `FORGED_SENDER` is caused by the way the mails are sent
* `AUTOGEN_PHP_SPAMMY` is caused by using phpMailer

But since `LOCAL_LINK_SHORT1` is giving the highest score, I would especially like to reduce that.

Looking forward to any questions, ideas and help!

Thanks a lot in advance, kind regards,
 Philipp

**Comment by moisseev** [comment]
There is no `LOCAL_LINK_SHORT1` symbol in the Rspamd default configuration.
You can use `grep` to find it in your customized config.

**Comment by pmamat** [comment]
Hi moisseev,
Thanks for your reply! That was really fast. 

I just checked out the rspamd repository to have a closer look at the default config. Now I am able to distinguish between default filters and filters coming from a custom config.

And I simply asked the server manager (hetzner Managed Server) about the custom filters. When I receive an answer, I will keep everybody informed for future reference.

Kind regards,
 Philipp

**Comment by pmamat** [comment, marked as answer]
Hi,

I received feedback from the server manager hetzner:

>    Symbol: LOCAL_LINK_SHORT1(2.30)

This rule applies if the body of the mail contains links from a URL shortener.
Using "goo.gl" was the cause. This service used to be a URL shortener and has not been active for some time. They have adjusted the filter rule.

>    Symbol: LOCAL_MASSMAIL_FIELD(0.90)

Is applied if the mail contains a "List-Unsubscribe" header, as this is an indicator for mass mails.

Hopefully this information will help others when searching for such symbols.

Kind regards,
 Philipp
